src/globus_cli/termio/context.py

```diff
@@ -68,5 +69,11 @@ def err_is_terminal():
     return sys.stderr.isatty()
 
 
-def term_is_interactive():
+def term_is_interactive() -> bool:
+    explicit_val = os.getenv("GLOBUS_CLI_INTERACTIVE")
+    if explicit_val is not None:
+        try:
+            return strtobool(explicit_val.lower())
+        except ValueError:
+            pass
```

Comment:
I wonder if we shouldn't just silently ignore misconfigured values here? I could imagine being confused by mismatching behavior because I accidentally typed "treu" or something.

Reply:
I wouldn't be opposed, as I agree that the result could be surprising. But I think this would require deeper restructuring.

We don't force this function to evaluate on every command, or at the start of a command.
So if `GLOBUS_CLI_INTERACTIVE=treu`, that would error... whenever `term_is_interactive()` gets called.
I don't like the idea that we could be halfway through something and then blow up.

If we added a step to our commands which always called some checker function, `validate_settings()`, then we could force `treu` to fail always, on all commands.

Let's write this up as an improvement, so we can look at how to attach a hook to the start of command execution (but not when `resilient_parsing` is set, so we don't break completion!).
This is the only variable which I know would be a candidate today, but we can look for `os.getenv` and `os.environ` usage to check for more.